dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -247,7 +247,7 @@ def _add_helper(self, other1, other2):
         if "num_negatives" in self.__calculations.keys():
             self.num_negatives = other1.num_negatives + other2.num_negatives
 
-    def _diff_helper(self, other_profile):
+    def diff(self, other_profile, options=None):
```

Comment:
ultimately renamed so the int column can just inherit this function instead of the int column creating its own. No reason for it to be private either.

Reply:
And allowing options for future use.